cmake/amr-wind-utils.cmake

```diff
@@ -71,6 +71,15 @@ macro(init_amrex)
   endif()
 endmacro(init_amrex)
 
+macro(init_amrex_hydro)
+  set(AMREX_HYDRO_SUBMOD_LOCATION "${CMAKE_SOURCE_DIR}/submods/AMReX-Hydro")
```

Comment:
What about support for external library?

Reply:
Upon further investigation, AMReX-hydro does not provide install rules and is configured as an `object` library only. As such it can only be used as a submodule within AMR-Wind.